.gitlab-ci.yml

```diff
@@ -146,19 +146,19 @@ variables:
   # To use images from datadog-agent-buildimages dev branches, set the corresponding
   # SUFFIX variable to _test_only
   DATADOG_AGENT_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BUILDIMAGES: v16742019-a2675dc
+  DATADOG_AGENT_BUILDIMAGES: v17115340-e510775
```

Comment:
Note: to be updated before merging (this isn't the latest `datadog-agent-buildimages` commit anymore).